www/survey.html

```diff
@@ -5,8 +5,9 @@
 <meta charset="UTF-8">
 <title>Caterpillars Count!</title>
 <script src="./js/jquery-1.11.3.min.js"></script>
-<script src="cordova.js"></script>
-<script src="./js/input-page.js"></script>
+<!--<script src="cordova.js"></script>-->
+<!--<script src="./js/input-page.js"></script>-->
```

Comment:
I see. maybe uncomment cordova.js? Even though it is not used, it's better to always include the framework js file I think.